values.yaml

```diff
@@ -116,3 +98,11 @@ commonLabels: {}
 
 # Global annotations to add to all resources
 commonAnnotations: {}
+
+# Shared volumes configuration
+volumes:
+  # Docker metadata volume shared between collector and beyla
+  dockerMetadata:
+    enabled: true
```

Comment:
I think I would disable the docker metadata enrichment in Kubernetes by default to save CPU. We should be using kubernetes labels for services in Kubernetes imho.

Anything we would need the metadata for in kubernetes? 🙏